gulpfile.js

```diff
@@ -37,22 +40,40 @@ gulp.task('hooks:precommit', ['build'], function() {
     .pipe(plugins.git.add());
 });
 
+gulp.task('analyze:node', [], function() {
+  var wconf = webpackConfig({
+    plugins: [new BundleAnalyzerPlugin({
+      analyzerMode: 'static',
+    })]
+  });
+
+  return gulp.src('src/index.js').pipe(webpack(wconf))
+});
+
+gulp.task('analyze:browser', ['lint:src'], function() {
+  var wconf = webpackConfig({
+    plugins: [new BundleAnalyzerPlugin({
+      analyzerMode: 'static',
+    })]
+  });
+
+  return gulp.src('src/browser.js').pipe(webpack(wconf))
+});
+
 gulp.task('build:node', ['lint:src'], function() {
     return gulp.src('src/**/*.js')
-        .pipe(plugins.babel())
+        .pipe(plugins.babel({
+          presets: ['env'],
+          plugins: [["transform-runtime", { "polyfill": false }]]
```

Comment:
Can you confirm `transform-runtime` plugin does not break React-based apps (fixed in #9)? I glanced at the [docs](https://babeljs.io/docs/plugins/transform-runtime/#technical-details) and it seems that it adds `babel-runtime` dependency that caused problems.

Reply:
For sure, I'll check to make sure.  The project I'm working on is using react, so we should have a test case to work against.

As far as #9, the transform-runtime plugin is the replacement of the babel-runtime to solve the issue you were running into.  https://github.com/babel/babel-loader#babel-is-injecting-helpers-into-each-file-and-bloating-my-code has some info.